csharp/test/Ice/interceptor/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.html
+*.xml
+*.xaml
```

Comment:
Do we really need this file? I don't see any of these files in the directory.

Reply:
No we don't I copied the test folder from Ice/operations and this come from there, I will remove it